_posts/2024-05-29-bundler-auto-install-just-got-a-whole-lot-better.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Bundler Auto-Install Just Got A Whole Lot Better
+layout: post
+author: Ngan Pham
+author_email: nganpham@gmail.com
+---
+
+Working in a large monolith with many engineers, you never fail to get a flurry of changes everytime you pull from `main`.
```

Comment:
Would it make sense to make short note at the beginning this is host blog post by XY from Z company sharing their experience in italic or so?